harness/determined/_data_layer/_data_layer.py

```diff
@@ -175,6 +176,13 @@ def cache_dataset(
         self._decorator_used = True
         dataset_version += "_train" if self._training else "_val"
 
+        # Track usage
+        # Temporary measure to determine whether to deprecate data layer/yogadl dependency
+        analytics.send_analytics(
+            "data_layer_cache_dataset",
+            {"dataset_version": dataset_version, "dataset_id": dataset_id},
```

Comment:
this additional data might not be necessary. is a binary used/not used metric enough for determining whether or not to deprecate yogadl/data layer?

Reply:
Personally I'd rather we not collect these bits of info as I think it's slightly invasive but without any clear benefit to our organization.

Reply:
makes sense. removed.